src/obs-plugin/plugin-main.cpp

```diff
@@ -86,24 +112,47 @@ struct obs_output_info virtualcam_output_info = {
 void start()
 {
     OBSData settings;
+    
     output = obs_output_create("virtualcam_output", "virtualcam_output", settings, NULL);
     obs_data_release(settings);
 }
 
+bool isSupportedResolution();
 bool obs_module_load(void)
 {
     blog(LOG_DEBUG, "VIRTUALCAM obs_module_load");
 
     QMainWindow* main_window = (QMainWindow*)obs_frontend_get_main_window();
+    //action = (QAction*)obs_frontend_add_tools_menu_qaction(obs_module_text("VirtualCamera_Start"));
     action = (QAction*)obs_frontend_add_tools_menu_qaction(obs_module_text("Start Virtual Camera"));
     auto menu_cb = []
     {
         if (obs_output_active(output)) {
+            //action->setText(obs_module_text("VirtualCamera_Start"));
             action->setText(obs_module_text("Start Virtual Camera"));
             obs_output_stop(output);
         } else {
-            action->setText(obs_module_text("Stop Virtual Camera"));
-            obs_output_start(output);
+            if(isSupportedResolution()){
+                //action->setText(obs_module_text("VirtualCamera_Stop"));
+                action->setText(obs_module_text("Stop Virtual Camera"));
+                obs_output_start(output);
+            } else {
+                
+                stringstream msg;
+                //msg << obs_module_text("UnsupportedResolution_Main") << endl;
```

Comment:
Thanks for taking a stab at localizations -- let's remove these comments for now and maybe track localization in a separate issue.